libraries/lib-datahandler/src/main/thrift/components.thrift

```diff
@@ -194,6 +196,8 @@ struct Release {
     54: optional COTSDetails cotsDetails,
 
     65: optional set<string> mainLicenseIds,
+    66: optional set<string> mainLicenseNames,
+    67: optional set<License> mainLicenses,
```

Comment:
Me neither. There is still a field "licenseNames" left, belonging to the releaseLink. But this one seems safe to delete.